core/modules/widgets/draggable.js

```diff
@@ -89,7 +92,7 @@ Selectively refreshes the widget if needed. Returns true if the widget or any of
 */
 DraggableWidget.prototype.refresh = function(changedTiddlers) {
 	var changedAttributes = this.computeAttributes();
-	if(changedAttributes.tag || changedAttributes["class"]) {
+	if(changedAttributes.tag || changedAttributes["class"] || changedAttributes.enabled) {
```

Comment:
I am noticing that our refresh handling is not complete from before. Could you change this line to:

if($tw.utils.count(changedAttributes > 0) {

so that we refresh when any of the attributes change.

Reply:
Sure thing